packages/account/src/providers/provider.ts

```diff
@@ -1539,22 +1558,24 @@ Supported fuel-core version: ${supportedVersion}.`
    * @param paginationArgs - Pagination arguments (optional).
    * @returns A promise that resolves to the balances.
    */
-  async getBalances(
-    owner: string | AbstractAddress,
-    paginationArgs?: CursorPaginationArgs
-  ): Promise<CoinQuantity[]> {
-    const result = await this.operations.getBalances({
-      first: 10,
-      ...paginationArgs,
+  async getBalances(owner: string | AbstractAddress): Promise<GetBalancesResponse> {
+    const {
+      balances: { edges },
+    } = await this.operations.getBalances({
+      /**
+       * The query parameters for this method were designed to support pagination,
+       * but the current Fuel-Core implementation does not support pagination yet.
+       */
```

Comment:
But it currently does not support pagination. If we pass pagination parameters, `fuel-core` throws an error.

Also, having this method receive pagination parameters would be misleading, as it won't use them.

Both methods that support pagination already (`getMessages` and `getCoins`) support being called without the pagination parameters.

I've already set this method to return `{ balance }` instead of `balances` to reduce the breaking change in the future.

Reply:
It looks like this may be a mistake on the `fuel-core` side.

The `getBalances` method should soon support pagination.